andela/ah-shakas-frontend
# #160577461 implement reading stats

## What does this PR do?
- include implementation for showing reading stats for a user
- include implementation to show author profile on article view

## Description of Task to be completed?
- Created stat component, container, reducer, and actions
- Added user profile to the article

## How should this be manually tested?
- clone this repo `git clone https://github.com/andela/ah-shakas-frontend.git`
- cd into ah-shakas-frontend `cd ah-shakas-frontend` 
- checkout to this branch `git checkout ft-reading-stats-1605774
61 `
- run `yarn install`
- run `yarn start`
- log in and visit your profile
- you should be able to see a stats tab that will show the statistics of all the articles you've published and their stats

## Any background context you want to provide?
- You should have the following in your .env
```
REACT_APP_BASE_URL=
REACT_APP_UPLOAD_BASE_URL=''
REACT_APP_CLOUD_NAME=''
REACT_APP_UPLOAD_PRESET=''
```
## Any background context you want to provide?

## What are the relevant pivotal tracker stories?
[ #160577461](https://www.pivotaltracker.com/n/projects/2198533/stories/160577461)
## Packages installed
None

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)


## Todos


## Screenshots (if appropriate)
<img width="1050" alt="image" src="https://user-images.githubusercontent.com/30701246/49226775-95796500-f3f8-11e8-87b6-a521a5b95bea.png">

<img width="1238" alt="image" src="https://user-images.githubusercontent.com/30701246/49226872-cf4a6b80-f3f8-11e8-93f2-15053fa7c795.png">

## Questions:
